SPECS/kured/kured.spec

```diff
@@ -0,0 +1,277 @@
+#
+# spec file for package kured
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+# nodebuginfo
+
+
+# Remove stripping of Go binaries.
+%define __arch_install_post export NO_BRP_STRIP_DEBUG=true
+# Project upstream commit.
+%define commit 685f328
+%global debug_package %{nil}
+Name:           kured
+Version:        1.6.1
+Release:        1.6%{?dist}
+Summary:        Kubernetes daemonset to perform safe automatic node reboots
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Management
+URL:            https://github.com/weaveworks/kured
+#Source0:       https://github.com/weaveworks/kured/archive/refs/tags/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-vendor.tar.gz
```

Comment:
Please provide instructions on how to generate the vendor tarball in a **reproducible** (see: `protobuf.spec` or `coredns-1.7.0.spec` from `1.0-dev` for instance) way, so that we can later recreate it and know how to generate a new one when we update the version.

In general we should never use a source file we don't know how to recreate in a consistent way or how to update it if needed.